packages/server/lib/browsers/browser-cri-client.ts

```diff
@@ -260,7 +260,8 @@ export class BrowserCriClient {
     debug('Target.attachedToTarget %o', event.targetInfo)
 
     try {
-      if (event.targetInfo.type !== 'page') {
+      // We don't track child tabs/page network traffic. 'other' targets can't have network enabled
+      if (event.targetInfo.type !== 'page' && event.targetInfo.type !== 'other') {
```

Comment:
Out of curiosity, have you found a place where all the possible values of `targetInfo.type` are enumerated/documented? It's unfortunate that [here](https://vanilla.aslushnikov.com/?Target.TargetInfo) it's only typed as a "string".

Reply:
I've found this: https://github.com/chromium/chromium/blob/main/content/browser/devtools/devtools_agent_host_impl.cc#L127-L140, but it unfortunately doesn't do a great job of indicating *what* each of those corresponds to.